packages/client/src/orders/types/postOrderShippingAddressChangeRequest.types.ts

```diff
@@ -0,0 +1,13 @@
+import type { Config } from '../../types/index.js';
+import type { Order, OrderShippingAddressChangeRequest } from './index.js';
+
+export type PostOrderShippingAddressChangeRequest = (
+  orderId: Order['id'],
+  data: OrderShippingAddressChangeRequestData,
+  config?: Config,
+) => Promise<number>;
+
+export type OrderShippingAddressChangeRequestData = Pick<
```

Comment:
Sorry, missed this one. Can you rename this to `PostOrderShippingAddressChangeRequestData` instead to be consistent, please?

Reply:
Done 👍